### ISSUE TITLE
Possible improvements to documentation

### ISSUE THREAD

**Issue by giordano** [JuliaHCI/ADI.jl#30]
After going through the documentation, I can give some recommendations:

1. the [introduction to HCI](https://juliahci.github.io/ADI.jl/dev/introduction/) is very interesting, it gives a nice overview of the topic without going in too many details.  As mentioned at the beginning, it isn't meant to be an exhaustive lecture note, but it may be good to leave some references at the end for further reading
2. the first snippets in the [getting started](https://juliahci.github.io/ADI.jl/dev/gettingstarted/) page don't show `using ADI`, which can confuse newcomers.  The first `using ADI` in the whole documentation is in the [Reduction Process](https://juliahci.github.io/ADI.jl/dev/gettingstarted/#Reduction-Process) section of the same page.  Maybe the home page might mention that you need to do `using ADI` to load it?
3. the [Decomposition](https://juliahci.github.io/ADI.jl/dev/gettingstarted/#Decomposition) section presents a formula without introducing the terms, especially `w` and `A`.  The page is geared towards people familiar with ADI, but briefly mentioning what these terms are could improve understanding of newcomers, too
4. the [reduction of β Pictoris](https://juliahci.github.io/ADI.jl/dev/examples/betapictoris/) example requires a bunch of packages which aren't necessarily present in the user's environment.  Maybe quickly mention that they might need to install them?
5. despite what is written in the [data reduction](https://juliahci.github.io/ADI.jl/dev/examples/betapictoris/#Data-Reduction) section, `mask_circle` doesn't seem to be re-exported by ADI.
   ```julia
   julia> mask_cube = mask_circle(cube, 10)
   ERROR: UndefVarError: mask_circle not defined
   Stacktrace:
    [1] top-level scope
      @ REPL[29]:1
   ```
   Same for `Kernels` and `construct` in the [contrast curve](https://juliahci.github.io/ADI.jl/dev/examples/betapictoris/#Contrast-Curve) section
6. [`decompose`](https://juliahci.github.io/ADI.jl/dev/api/#ADI.decompose) docstring only shows the syntax of the function without further details.  Is it supposed to not give more details?
7. the documentation lacks community guidelines, like instructions about how to contribute, report issues, seek support

**Comment by mileslucas** [comment]
Thanks for all the in-depth comments and suggestions, I really appreciate them. I've made a slew of documentation revisions; thanks for pointing out locations lacking in context, especially.

Your point 5 may be due to mismatched docs and code- I didn't start re-exporting HCIToolbox until version `0.5.3` (which is the master branch), so if you want to try again I'd recommend doing a fresh pull.

For point 7 I've added contribution and support section to the README and homepage of the docs.

Again, thanks for all the help!
